### ISSUE TITLE
Add support for Swift Package Manager

### ISSUE THREAD

**Issue by LordZardeck** [Timing-GmbH/DateRangePicker#15]
Swift Package Manager is integrated into Xcode 11, and it would be nice for me to move to that from cocoapods. Are there plans to add support? I think a valid `Package.swift` is all that's needed

**Comment by MrMage** [comment]
That should indeed be easy. Do you want to file a PR?

**Comment by LordZardeck** [comment]
I actually looked into this, and it's harder than you would expect, since this project uses resources, and you have to specifically support that. Especially xib files. So I'm guess support isn't going to come soon.

**Comment by MrMage** [comment]
Right, I did not think of that. I think SPM now supports resources as of Swift 5.3, though: https://useyourloaf.com/blog/add-resources-to-swift-packages/. Haven't looked into it myself, though.

**Comment by LordZardeck** [comment]
Yes, but it requires you to change your bundle loading to use Bundle.main which will break cocoa pods

**Comment by MrMage** [comment]
I guess we could make that a compile-time or runtime option, though. Or just use `Bundle(forClass:)`.
